api/api.go

```diff
@@ -108,5 +108,7 @@ func (api *API) enablePrivateEndpoints() {
 	r.Put("/filter-outputs/{id}", api.putFilterOutput)
 	r.Post("/filter-outputs/{id}/events", api.addFilterOutputEvent)
 
+	r.Get("/datasets/{dataset_id}/editions/{edition}/versions/{version}/json", api.getDatasetJSONHandler)
```

Comment:
If the endpoint is now meant to be available to publishing then some secrets need to be updated for dp-api-router

Reply:
Create a [PR](https://github.com/ONSdigital/dp-configs/pull/1131) to address this issue